packages/web-components/tests/e2e-storybook/cypress/integration/masthead/masthead-scoped-search.e2e.js

```diff
@@ -1,148 +1,143 @@
-
-    cy.get('dds-masthead > dds-search-with-typeahead')
-      .shadow()
-      .find('input[type="text"]')
-      .type('cloud', { force: true });
-
-    cy.get('dds-search-with-typeahead-item').should('have.length', 10);
-
-    cy.takeSnapshots();
-  });
-
-  it('should retrieve less results with "pw" scope', () => {
-    // Mock scoped search typeahead API
-    cy.intercept('**/search/typeahead/v1?*', {
-      fixture: 'scoped-typeahead.json',
-    });
-
-    cy.get('dds-masthead > dds-search-with-typeahead')
-      .shadow()
-      .find('.bx--header__search--search')
-      .click();
-
-    cy.get('dds-scoped-search-dropdown')
-      .shadow()
-      .find('.bx--dropdown')
-      .click();
-
-    cy.get('dds-scoped-search-dropdown')
-      .find(`bx-dropdown-item[value="pw"]`)
-      .click();
-
-    cy.get('dds-scoped-search-dropdown').should('have.value', 'pw');
-
-    cy.get('dds-masthead > dds-search-with-typeahead')
-      .shadow()
-      .find('input[type="text"]')
-      .type('cloud', { force: true });
-
-    cy.get('dds-search-with-typeahead-item').should('have.length', 5);
-    cy.takeSnapshots();
-  });
-});
-
-describe('dds-masthead | scoped search (mobile)', () => {
-  beforeEach(() => {
-    cy.visit(`/${_pathScopedSearch}`);
-    cy.viewport(320, 780);
-
-    cy.waitUntil(() => cy.get('[data-autoid="dds--masthead-default__l0-nav0"]').should('not.be.empty'));
-  });
-
-  it('should open the search bar on click', () => {
-    cy.get('dds-masthead > dds-search-with-typeahead')
-      .shadow()
-      .find('.bx--header__search--search')
-      .click();
-
-    cy.takeSnapshots();
-  });
-
-  it('should retrieve regular results with "all" scope', () => {
-    cy.get('dds-masthead > dds-search-with-typeahead')
-      .shadow()
-      .find('.bx--header__search--search')
-      .click();
-
-    cy.get('dds-scoped-search-dropdown-mobile').should('have.value', 'all');
-
-    cy.get('dds-masthead > dds-search-with-typeahead')
-      .shadow()
-      .find('input[type="text"]')
-      .type('cloud', { force: true });
-
-    cy.get('dds-search-with-typeahead-item').should('have.length', 10);
-
-    cy.takeSnapshots();
-  });
-
-  it('should retrieve less results with "pw" scope', () => {
-    // Mock scoped search typeahead API
-    cy.intercept('**/search/typeahead/v1?*', {
-      fixture: 'scoped-typeahead.json',
-    });
-
-    cy.get('dds-masthead > dds-search-with-typeahead')
-      .shadow()
-      .find('.bx--header__search--search')
-      .click();
-
-    cy.get('dds-scoped-search-dropdown-mobile')
-      .shadow()
-      .find('.bx--select-input')
-      .select('pw');
-    cy.get('dds-scoped-search-dropdown-mobile').should('have.value', 'pw');
-
-    cy.get('dds-masthead > dds-search-with-typeahead')
-      .shadow()
-      .find('input[type="text"]')
-      .type('cloud', { force: true });
-
-    cy.get('dds-search-with-typeahead-item').should('have.length', 5);
-    cy.takeSnapshots();
-  });
-});
+// /**
```

Comment:
Should we just remove this file for now?

Reply:
I commented it out just to make it easier to bring back later, but yeah seems we have some prettier issues anyway. I'll delete instead